### PR TITLE
Added 'News' section + new BP 4.2

### PR DESCRIPTION
**Pull request by i0natan**
To motivate people to re-visit, I've added a 'news box' in which will show the latest best practices and various news. The monthly maintainer will update this section + hopefully, others will update frequently during the month.

Also had to add a new BP (4.2) so the news box refer to... new content

@MattJin Any chance to update the corresponding local branches?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -411,15 +420,27 @@ All statements above will return false if used with `===`
 
 <br/><br/>
 
-## ![✔] 4.2 Detect code issues with a linter
+## ![✔] 4.2 Include 3 parts in each test name
+
+**TL;DR:** Make the test speak at the requirements level so it's self explanatory also to QA engineers and developers who are not familiar with the code internals. State in the test name what it being tested (unit under test), under what circumstances and what is the expected result
```
Comment (BrunoScheufler) [inline README.md]: I think there's a small typo in "State in the test name what it being tested..." -> "State in the test name what is being tested"

Comment (i0natan) [reply README.md]: @BrunoScheufler Yeah, good catch

Comment (i0natan) [reply README.md]: @BrunoScheufler Fixed, please approve

**Comment by i0natan** [comment]
@js-kyle You didn;t approve but I'm not sure what is your feedback?

**Comment by js-kyle** [comment]
@i0natan sorry not sure what went wrong there, I meant to approve initially

**Pull request merged**